web/Makefile

```diff
@@ -9,20 +9,52 @@ NOTEBOOK_OUTPUT_DIR := docs/try-vast
 NOTEBOOK_INPUTS := $(wildcard $(NOTEBOOK_INPUT_DIR)/*.md)
 NOTEBOOK_OUTPUTS := $(addprefix $(NOTEBOOK_OUTPUT_DIR)/, $(notdir $(NOTEBOOK_INPUTS)))
 
-all: changelog notebooks
+DOT_GITHUB_OUTPUT_DIR := docs/develop-vast/contributing
 
-changelog: $(CHANGELOG_OUTPUT)
-
-notebooks: $(NOTEBOOK_OUTPUTS)
+all: changelog notebooks dot-github
 
-clean:
-	rm -f $(CHANGELOG_OUTPUT)
-	$(MAKE) -C $(NOTEBOOK_INPUT_DIR) clean
-	git clean -f -d -x $(NOTEBOOK_OUTPUT_DIR)
+changelog: $(CHANGELOG_OUTPUT)
 
 $(CHANGELOG_OUTPUT): $(CHANGELOG_INPUT)
 	cp -f $< $@
 
+notebooks: $(NOTEBOOK_OUTPUTS)
+
 $(NOTEBOOK_OUTPUTS): $(NOTEBOOK_INPUTS)
 	@$(MAKE) -C $(NOTEBOOK_INPUT_DIR) md
 	cp -r $(NOTEBOOK_INPUT_DIR)/output/md/* $(NOTEBOOK_OUTPUT_DIR)
+
+.github:
+	git clone https://github.com/tenzir/.github.git
+	cd .github && git pull
+
+dot-github: \
+  $(DOT_GITHUB_OUTPUT_DIR)/code-of-conduct.md \
+  $(DOT_GITHUB_OUTPUT_DIR)/coding-style.md \
+  $(DOT_GITHUB_OUTPUT_DIR)/security.md \
+  $(DOT_GITHUB_OUTPUT_DIR)/workflow.md
+
+$(DOT_GITHUB_OUTPUT_DIR)/code-of-conduct.md: .github
+	@echo "---\nsidebar_position: 0\n---\n" > $@
+	@cat .github/$(@F) >> $@
+
+$(DOT_GITHUB_OUTPUT_DIR)/coding-style.md: .github
+	@echo "---\nsidebar_position: 2\n---\n" > $@
+	@cat .github/$(@F) >> $@
+
+$(DOT_GITHUB_OUTPUT_DIR)/security.md: .github
+	@echo "---\nsidebar_position: 10\n---\n" > $@
+	@cat .github/$(@F) >> $@
+
+$(DOT_GITHUB_OUTPUT_DIR)/workflow.md: .github
+	@echo "---\nsidebar_position: 1\n---\n" > $@
+	@cat .github/$(@F) >> $@
+
+clean:
+	rm -f $(CHANGELOG_OUTPUT)
+	$(MAKE) -C $(NOTEBOOK_INPUT_DIR) clean
+	git clean -f -d -x $(NOTEBOOK_OUTPUT_DIR)
+	rm $(DOT_GITHUB_OUTPUT_DIR)/code-of-conduct.md
+	rm $(DOT_GITHUB_OUTPUT_DIR)/coding-style.md
+	rm $(DOT_GITHUB_OUTPUT_DIR)/security.md
+	rm $(DOT_GITHUB_OUTPUT_DIR)/workflow.md
```

Comment:
Why not just use `git clean` as well?
```
git clean -fd $(DOT_GITHUB_OUTPUT_DIR) 
```

If there's a good reason against, it would probably make sense to define a variable so you don't have to keep them in sync:
```
DOT_GITHUB_OUTPUT_FILES=  $(DOT_GITHUB_OUTPUT_DIR)/code-of-conduct.md \
  $(DOT_GITHUB_OUTPUT_DIR)/coding-style.md \
  $(DOT_GITHUB_OUTPUT_DIR)/security.md \
  $(DOT_GITHUB_OUTPUT_DIR)/workflow.md
```

Also, I think this also needs to remove the `.github` directory.

Reply:
The "good reason against" was mostly to not wipe somebody's working tree where they have something uncommitted. The `git clean` approach deletes that without further ado.

What you deem that as "good reason" or being overly cautious?

Reply:
Yeah, sounds reasonable to me.